spanner/api/Spanner.Samples.Tests/CopyBackupTest.cs

```diff
@@ -38,9 +38,11 @@ public void CopyBackup()
         string target_backupId = SpannerFixture.GenerateId("test_", 16);
         DateTimeOffset expireTime = DateTimeOffset.UtcNow.AddHours(12);
 
-        Backup backup = copyBackupSample.CopyBackup(source_Instance_id, source_Project_id, source_backupId, 
-            target_Instance_id, target_Project_id, target_backupId, expireTime);
+        Backup backup = copyBackupSample.CopyBackup(
```

Comment:
The variables are still declared above, just not used any more.

Reply:
Done, thanks!